### PR TITLE
New Boxtribute theme

### PR DESCRIPTION
**Pull request by DurkoMatko**
change: styling to fit new Boxtribute theme

### CODE REVIEW

**Comment by DurkoMatko** [comment]
[Unify UI theming in preparation for rebranding #large](https://trello.com/c/on228c5S/476-unify-ui-theming-in-preparation-for-rebranding-large)

File: assets/css/custom.css [hunk 1]
```diff
@@ -795,45 +804,34 @@ div.filter-applied {
...
+	box-shadow: 8px 0px var(--stockOverviewLevel4) inset;
 	padding-left: 16px;
 }
 .overview-level-3 td:last-child{
-	box-shadow: -8px 0px #FFAD63 inset;
+	box-shadow: -8px 0px var(--stockOverviewLevel4) inset;
 	padding-right: 16px;
-}
-
-.overview-filter-gender > div.btn.btn-sm.btn-default{
```
Comment (HaGuesto) [inline assets/css/custom.css:830]: these css parts I added recently for the stock overview to connect the color of the levels with the color of the filters. I sther a reason why you removed it @DurkoMatko ?

File: assets/css/style.css [hunk 2]
```diff
@@ -431,7 +454,7 @@ body > iframe { display: none; }
 	background-image: -moz-linear-gradient(top,#f8f8f8,#f1f1f1);
 	background-image: -ms-linear-gradient(top,#f8f8f8,#f1f1f1);
 	border: 1px solid #c6c6c6;
-	color: #333;
+	/* color: #333; */
```
Comment (HaGuesto) [inline assets/css/style.css:457]: remove comment

**Comment by HaGuesto** [comment]
## more things I found
- The titles need more space to the menu
image

- the form field have round corners and old blue when clicking into them (focus)
image

- the tabs for forms look different than for lists
image
image

**Comment by aerinsol** [comment]
@HaGuesto I don't agree. The header image is not suitable for login as it
serves a different purpose. Will add that and all other needed changes to
Trello card directly.

**Comment by aerinsol** [comment]
Three dots is on purpose. Common to use this in modern design language.

File: assets/css/custom.css (hunk 1 above)
Comment (aerinsol) [reply assets/css/custom.css:830]: I asked him to, as it wasn't clear that's what it did.

**Comment by DurkoMatko** [comment]
_I agree with this actually, but it's of lower importance since this is the "desktop" app and I don't want to waste @DurkoMatko's time on that. Rather than putting it on two separate lines, I would delete "BoxAid" and just show the base._
- deleted orgname

_the behavior of filters in beneficiaries and boxes do not feel right.
the blue when hovering over them seems wrong. The edges are also round
where other buttons have a sharper edge._
- removed onHover behavior

_the left-side menu should not have a scrollbar for lef-right--> the
space for the menu should be increased._
- can't reproduce

_ there is no way to get to the start page anymore. Before there was
always a link on the logo in the header menu to go back to the start page_
-done

File: assets/css/style.css (hunk 2 above)
Comment (DurkoMatko) [reply assets/css/style.css:457]: done

File: assets/css/style.css [hunk 3]
```diff
@@ -94,10 +103,10 @@ p{
 	margin-top: -5px;
 	margin-bottom: -5px;
 }
-.header-top-inner .nav>li>a:hover, .header-top-inner .nav>li>a:focus, .header-top-inner .nav .open>a, .header-top-inner .nav .open>a:hover, .header-top-inner .nav .open>a:focus{
+/* .header-top-inner .nav .open>a:hover, .header-top-inner .nav .open>a:focus{
```
[the remark this answers is not in the document]
Comment (DurkoMatko) [reply assets/css/style.css:106]: done

**Comment by aerinsol** [comment]
> 
> _the form field have round corners and old blue when clicking into them (focus)_
> 
> * this was requested from Roanna (inputs on focus/active should have this blue outline)

I think it's the new light blue, not the old light blue.

> _the tabs for forms look different than for lists_
> 
> * this was the case in the old design as well. They are 2 completely different controls. What should I do? @aerinsol  :
>   image
>   image

@HaGuesto and @DurkoMatko leave it. We're not trying to massively improve the functionality of the old app at this point, just be consistent with the rebrand.

> _I would adjust the font-style for fancy-graphs and the start page. They seem to large now in comparison to other text and I do not know about the gray._
> 
> * Do u approve? @aerinsol
@DurkoMatko @HaGuesto can someone share me a screencap?

File: assets/css/custom.css (hunk 1 above)
Comment (HaGuesto) [reply assets/css/custom.css:830]: So, can we add this again ?

Comment (HaGuesto) [reply assets/css/custom.css:830]: @aerinsol

**Comment by aerinsol** [comment]
I would say the following points.
- Ignore the header overflow. It's completely unimportant.
- leave the stock filter as is. Earlier behavior of color was not clear. If
someone complains, let me know and assign it to me.
- leave the color of text the start page.
- SECTION HEADERS should be black, not gray.

**Comment by HaGuesto** [comment]
Thanks @aerinsol!
I think the last steps for you @DurkoMatko are then:

- [ ]  Fix search button

- [ ] SECTION HEADERS should be black, not gray. (in the menu)

(- exchange the favicon if possible)

**Comment by DurkoMatko** [comment]
@HaGuesto done those 3

**Pull request merged**